README.md

```diff
@@ -140,20 +140,19 @@ pipeline {
     agent any
     stages {
         stage("synopsys-security-scan") {
+            when {
+                anyOf {
+                    branch 'master'
+                    branch pattern: "PR-\\d+", comparator: "REGEXP"
```

Comment:
If this restriction is only for PRs to scan in this stage, then this whole line can be simplified to - `changeRequest()`

```
when {
       anyOf {
             branch 'master'
             changeRequest()
      }
 }
```